scripts/Dockerfile.rembg

```
@@ -0,0 +1,4 @@
+FROM python:3.11-slim
+RUN pip install --no-cache-dir 'rembg[cpu,cli]'
```

Hadolint (2.14.0)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)